roofit/hs3/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 1995-2019, Rene Brun and Fons Rademakers.
+# All rights reserved.
+#
+# For the licensing terms see $ROOTSYS/LICENSE.
+# For the list of contributors see $ROOTSYS/README/CREDITS.
+
+############################################################################
+# CMakeLists.txt file for building ROOT rootfit/histfactory package
+# @author Carsten Burgard, CERN
+############################################################################
+
+ROOT_STANDARD_LIBRARY_PACKAGE(HS3
+  HEADERS
+  JSONInterface.h
+  RooJSONFactoryWSTool.h
+  HistFactoryJSONTool.h
+  JSONParser.h
+  RYMLParser.h
+  SOURCES
+  src/RooJSONFactoryWSTool.cxx
+  src/HistFactoryJSONTool.cxx
+  src/JSONParser.cxx
+  src/RYMLParser.cxx
+  src/JSONFactories_RooFitCore.cxx
+  src/JSONFactories_HistFactory.cxx
+  DICTIONARY_OPTIONS
+    "-writeEmptyRootPCM"
+  DEPENDENCIES
+    RooFit
+    RooFitCore
+    RooStats
+    HistFactory    
+)
+
+if(yml)
```

Comment:
Please move it in one level higher in `roofit/CMakeLists.txt ` and add CMake option `yml` (exactly the same way how it is done for `xml`)